src/Microsoft.AspNetCore.CookiePolicy/SameSitePolicy.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.CookiePolicy
+{
+    public enum SameSitePolicy
+    {
+        None = 0,
+        LaxOrStrict,
```

Comment:
Names need work. The policy specifies the minimum required level. Maybe you don't need a new enum, you can use the existing one and just name the option SameSiteMinimum.